perf/terraform/terraform.tf

```
@@ -2,37 +2,78 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.0"
+      version = "4.67.0"
     }
   }
 }
 
-module "server_region" {
-  source = "./region"
-  region = "us-west-2"
-  ami = "ami-0747e613a2a1ff483"
-
-  common_tags = {
+locals {
+  tags = {
     Project = "perf"
   }
+  root = "."
 }
 
-module "client_region" {
-  source = "./region"
-  region = "us-east-1"
-  ami = "ami-06e46074ae430fba6"
+variable "common_enabled" {
```

```suggestion
variable "cleanup_automation_enabled" {
```

Nit pick: This would be more descriptive to me.